MeshLib/MeshGenerators/VtkMeshConverter.cpp

```diff
@@ -250,7 +260,14 @@ MeshLib::Mesh* VtkMeshConverter::constructMesh(const double* pixVal,
 	if (elements.empty())
 		return nullptr;
 
-	return new MeshLib::Mesh("RasterDataMesh", nodes, elements); // the name is only a temp-name, the name given in the dialog is set later
+	if (value_vec->empty())
+		properties.removePropertyVector("Colour");
+
+	boost::optional< MeshLib::PropertyVector<unsigned>& > materials =
+		properties.createNewPropertyVector<unsigned>("MaterialIDs", MeshLib::MeshItemType::Cell, 1);
+	materials->insert(materials->begin(), elements.size(), 0);
```

Comment:
What is the size of `*materials` vector after the creation?
 - If it is 0, then the insert is OK, though `std::fill_n(std::back_inserter(*materials), elements.size(), 0)` is better, because the intention is to fill the vector, not to insert zeros in front of the existing values.
 - If it is not 0, then the insert is probably wrong here, because the existing elements would be preserved and the resulting  `*materials` vector would grow by `elements.size()`.